src/components/FileTree/index.spec.tsx

```diff
@@ -252,14 +252,6 @@ describe(__filename, () => {
       expect(isNodeExpanded(node)).toBeFalsy();
     });
 
-    it('logs a warning message when no version is loaded', () => {
```

Comment:
Removing this is fine by me. The execution of the if statement is already covered by five other tests.